<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>openPilotLog v0.6</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewFlight">
   <property name="text">
    <string>NewFlight</string>
   </property>
   <property name="toolTip">
    <string>Add a new flight to the logbook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Change Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLogbook">
   <property name="text">
    <string>Logbook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAircraft">
   <property name="text">
    <string>Aircraft</string>
   </property>
   <property name="toolTip">
    <string>Show or edit your aircraft</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionNewAircraft">
   <property name="text">
    <string>NewAircraft</string>
   </property>
   <property name="toolTip">
    <string>Add a new aircraft to the database</string>
   </property>
  </action>
  <action name="actionPilots">
   <property name="text">
    <string>Pilots</string>
   </property>
   <property name="toolTip">
    <string>Show or edit Pilots</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNewPilot">
   <property name="text">
    <string>NewPilot</string>
   </property>
   <property name="toolTip">
    <string>Add a new Pilot to the database</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="toolTip">
    <string>Create or restore a backup of the database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
